templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -52,19 +52,18 @@ If you have Homebrew installed and prefer to use it, here's how.
 
 1. Set up the Zulip Desktop apt repository and its signing key, from a
    terminal:
+        
+        sudo apt-key adv --keyserver pool.sks-keyservers.net --recv 69AD12704E71A4803DCA3A682424BE5AE9BD10D9
 
-```
-sudo apt-key adv --keyserver pool.sks-keyservers.net --recv 69AD12704E71A4803DCA3A682424BE5AE9BD10D9
-echo "deb https://dl.bintray.com/zulip/debian/ stable main" | \
-  sudo tee -a /etc/apt/sources.list.d/zulip.list
-```
+        echo "deb https://dl.bintray.com/zulip/debian/ stable main" | \
+
+        sudo tee -a /etc/apt/sources.list.d/zulip.list
 
 2. Install the client, from a terminal:
-```
-sudo apt update
-sudo apt install zulip
-```
+        
```

Comment:
@sagar-kalra shouldn't the blockquotes be required to signify that the blocks are code?

Reply:
no
I learnt this from the article you provided the link to.
And this is working you can check the screenshot of it.

Reply:
Strange that it is working tho.

https://gist.githubusercontent.com/clintel/1155906/raw/71b2ab2e4444cb826b45f9988d067d1dd1edcc2c/gistfile1.md,

1. This is a numbered list.
2. I'm going to include a fenced code block as part of this bullet:

    ```
    Code
    More Code
    ```

Reply:
Note: I use backticks for the fenced block above.

Reply:
https://gist.github.com/dergachev/97ba9add0e27cac38990
Check this out @rht

Reply:
Agreed it works even without the backticks.